clusterloader2/pkg/config/cluster.go

```diff
@@ -50,13 +50,12 @@ type PrometheusConfig struct {
 	ScrapeKubeProxy    bool
 }
 
-// GetMasterIp returns the first master ip, added for backward compatibility.
-// TODO(mmatt): Remove this method once all the codebase is migrated to support multiple masters.
-func (c *ClusterConfig) GetMasterIp() string {
+// GetMasterIps returns all registered master IPs.
+func (c *ClusterConfig) GetMasterIps() []string {
 	if len(c.MasterIPs) > 0 {
```

Comment:
Once we get rid of all the callers of this method we won't need it anymore. You can just delete it and get masterIps directly from the MasterIPs field in the downstream code.